appium/lib/screen-object/accessors/element.rb

```diff
@@ -174,14 +183,15 @@ def swipe_element_right
         swipe_screen_element(:right, 2000)
       end
 
-      def scroll_for_dynamic_element_click (expected_text)
+      def scroll_for_dynamic_element_tap (expected_text)
```

Comment:
`scroll_for_dynamic_element_tap` should actually be called `scroll_dynamic_text_to_tap` and remove the unwanted  space between method name and arguments

Reply:
Do we even need this method though, people can simply call the method to scroll to dynamic text and then tap in their code rather then us creating methods with all these combinations